tests/unittests/OperatorTest.cpp

```diff
@@ -928,21 +928,30 @@ TEST_P(InterpAndCPU, ScatterAssignQuantized) {
 }
 
 TEST_P(Operator, QuantizeAndDequantize) {
-  Tensor inputs(ElemKind::FloatTy, {1, 4});
-  inputs.getHandle() = {1, 1.2f, 0.5f, 1.3f};
+  Tensor input1(ElemKind::FloatTy, {1, 4});
+  Tensor input2(ElemKind::FloatTy, {1, 4});
+  input1.getHandle() = {1, 1.2f, 0.5f, 1.3f};
+  input2.getHandle() = {1, 1.2f, 0.5f, 1.3f};
 
   auto *A = mod_.createVariable(ElemKind::FloatTy, {1, 4}, "A",
```

Comment:
super nit: you can remove Tensor input1 and just directly initialize variable A,
A->getPayload().getHandle()={1, 1.2f, 0.5f, 1.3f}.
And if so when you run E_.run just omit additional initialization of tensors.